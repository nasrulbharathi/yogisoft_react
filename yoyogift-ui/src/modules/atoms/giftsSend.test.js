import React from "react";
import { shallow } from "enzyme";
import { GiftsSend } from "./GiftsSend";

describe("Gifts Send ", () => {
  it("should be rendered", () => {
    const dataValue = [
      {
        cardName: "testcard",
        cardPoints: 123,
        receiverEmail: "test@test.com",
        cardIssueDate:
          "Mon May 04 2020 14:51:28 GMT+0530 (India Standard Time)",
        cardExpiryDate:
          "Mon Aug 04 2020 14:51:28 GMT+0530 (India Standard Time)",
      },
    ];
    let p = {
      classes: {},
      data: dataValue,
    };
    const wrapper = shallow(<GiftsSend {...p} />);
    expect(wrapper).toMatchSnapshot();
  });
});
