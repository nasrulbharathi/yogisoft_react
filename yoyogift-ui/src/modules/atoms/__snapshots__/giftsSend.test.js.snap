// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gifts Send  should be rendered 1`] = `ShallowWrapper {}`;
